### DISCUSSION TITLE
How to disable pop up message about custom prefix

### DISCUSSION THREAD

**Discussion by cat-roll** [jagrosh/MusicBot#1519] [Questions]
Basically this never happen when i use 0.3.9v even with custom prefix.
Is there any way to disable the pop up?
![Screenshot 2024-03-18 114930](https://github.com/jagrosh/MusicBot/assets/147711447/e1dd892d-c462-4f53-9a66-2a57bdb3ff6c)

**Comment by jagrosh** [comment]
This will be fixed in 0.4.1 (see #1521)
